src/main.rs

```diff
@@ -154,14 +158,26 @@ async fn transcode(mut cmd: TranscodeCommand) -> anyhow::Result<()> {
     ))?;
 
     for url in cmd.urls.clone() {
-        handle_url(
+        let result = handle_url(
             url.as_str(),
             &term,
             &mut api,
             cmd.clone(),
             index_response.passkey.clone(),
         )
-        .await?;
+        .await;
+
+        match result {
```

Comment:
Could you instead not handling success and handling error just check for error with 
```Rust
if let Err(e) = result {
}
```